### PR TITLE
Re-render for updated build via grayskull, add maintainer

### LINKED ISSUE #6: Add support for grpcio
- [ ] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

### PR DESCRIPTION
**Pull request by asford**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

The current feedstack is broken due to an invalid, older entrypoint that was updated at upstream v2.
Re-renders feedstock via grayskull to take updated python version requirements and fix entrypoints.
I've added myself to the maintainers list in case this causes further breaking issues.
closes #6

### CODE REVIEW

**Comment by asford** [comment]
@conda-forge-admin, please rerender

**Comment by asford** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,45 +1,48 @@
...
+    - cat /dev/null | protoc-gen-mypy
+    - cat /dev/null | protoc-gen-mypy_grpc
+  requires:
+    - pip
 
 about:
   home: https://github.com/dropbox/mypy-protobuf
+  summary: Generate mypy stub files from protobuf specs
   license: Apache-2.0
-  license_family: Apache
```
Comment (hmaarrfk) [inline recipe/meta.yaml:39]: Is license family no longer necessary?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,45 +1,48 @@
...
 test:
   imports:
     - mypy_protobuf
+    - mypy_protobuf.extensions_pb2
   commands:
-    - which protoc-gen-mypy
+    - cat /dev/null | protoc-gen-mypy
+    - cat /dev/null | protoc-gen-mypy_grpc
+  requires:
+    - pip
```
Comment (hmaarrfk) [inline recipe/meta.yaml:37]: Why do you require pip for tests but not run?

**Comment by hmaarrfk** [comment]
Just a few small questions

File: recipe/meta.yaml (hunk 1 above)
Comment (asford) [reply recipe/meta.yaml:39]: I believe this is no longer needed as license is a [spdx identifier](https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx).

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,45 +1,48 @@
...
+    - python >=3.6
 
 test:
   imports:
     - mypy_protobuf
+    - mypy_protobuf.extensions_pb2
   commands:
-    - which protoc-gen-mypy
+    - cat /dev/null | protoc-gen-mypy
+    - cat /dev/null | protoc-gen-mypy_grpc
```
Comment (asford) [inline recipe/meta.yaml:35]:
```suggestion
    - cat /dev/null | protoc-gen-mypy_grpc
    - pip check
```

File: recipe/meta.yaml (hunk 2 above)
Comment (asford) [reply recipe/meta.yaml:38]: Good catch, sorry. `grayskull` normally adds `pip check` to the tests, which I've re-added here.

**Comment by hmaarrfk** [comment]
I'm curious why this closes #6. It seems that there are a few dependencies to get this to work right? We didn't add any.

**Pull request merged**